model/search_pipeline/create_search_pipeline/operations.smithy

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/latest/search-plugins/search-pipelines/creating-search-pipeline/"
+)
+
+@xOperationGroup("search_pipeline.create_search_pipeline")
```

Comment:
`search_pipeline.create` is more appropriate because the namespace `search_pipeline` already gives you a context of what being created. (Just like we have `indices.create` and such)

Reply:
Thank you, I got it